wiki/missionmaker/modules.md

```diff
@@ -211,12 +203,24 @@ This module allows you to tweak the settings for player names tags.
 
 Name | Type | Description | Default value
 ---- | ---- | ---- | ---- |
-Show player names | Option | Let you choose when nametags appears. | "Do Not Force"
+Show player names | Option | Let you choose when nametags appears. | Do Not Force
 layer Names View Distance | Number | Distance (in meters) at which player names are shown. | 5
-Show name tags for AI? | Option | Show the name and rank tags for friendly AI units, or by default allows players to choose it on their own. | "Do Not Force"
-Show crew info? | Option | Show vehicle crew info, or by default allows players to choose it on their own. | "Do Not Force"
+Show name tags for AI? | Option | Show the name and rank tags for friendly AI units, or by default allows players to choose it on their own. | Do Not Force
+Show crew info? | Option | Show vehicle crew info, or by default allows players to choose it on their own. | Do Not Force
 Show for Vehicles? | Boolean | Show cursor NameTag for vehicle commander (only if client has name tags enabled). | No
 
+### 1.16 Pointing settings
+*Part of ace_finger*
+
+This module allow you to tweak settings for finger pointing such as is if it's enabled or the distance people can see you point things
```

Comment:
"This module allows you ..."
"... as if it's ..."
"... the distance at which people can see you pointing at something."